### PR TITLE
tools: update query-status descriptions of dm

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? 

This PR updates the `query-status` descriptions of DM.

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1994

### Which version does your change affect? 

dev, v2.1, v3.0, v3.1
I'll **update v2.1, v3.0 and v3.1 after 2 LGTMs**.

### CODE REVIEW

File: dev/how-to/migrate/from-aurora.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ black-white-list:
 # Mydumper global configuration. Each instance can quote it by the configuration item name.
 mydumpers:
   global:
-    extra-args: "-B test_db -T test_table"  # Only outputs the `test_table` table of the `test_db` schema and can configure any parameter of Mydumper.
+    extra-args: "-B test_db -T test_table"  # The extra Mydumper argument. Starting from DM 1.0.2, DM automatically generates the "table-list" option. For versions earlier than v1.0.2, you need to configure the option manually.
```
Comment (csuzhangxc) [inline dev/how-to/migrate/from-aurora.md]: `"table-list"` or `"tables-list"` or `tables-list`.  @lichunzhu 

```
-T, --tables-list           Comma delimited table list to dump (does not exclude regex option)
```

Comment (lichunzhu) [reply dev/how-to/migrate/from-aurora.md]: I think tables-list is better. How about using bold text, like **tables-list**?

Comment (csuzhangxc) [reply dev/how-to/migrate/from-aurora.md]: how about using ``` `tables-list` ```?

Comment (lichunzhu) [reply dev/how-to/migrate/from-aurora.md]: OK, I think ``` `tables-list` ``` is better.

Comment (lichunzhu) [inline dev/how-to/migrate/from-aurora.md]:
```suggestion
    extra-args: "-B test_db -T test_table"  # The extra Mydumper argument. Starting from DM 1.0.2, DM automatically generates the `tables-list` option. For versions earlier than v1.0.2, you need to configure the option manually.
```

File: dev/reference/tools/data-migration/deploy.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ black-white-list:
 mydumpers:
   global:
     mydumper-path: "./bin/mydumper"   # The file path of the Mydumper binary.
-    extra-args: "-B test_db -T test_table"  # Only dumps the "test_table" table of the "test_db" database. It can configure any Mydumper argument.
+    extra-args: "-B test_db -T test_table"  # The extra Mydumper argument. Starting from DM 1.0.2, DM automatically generates the "table-list" option. For versions earlier than v1.0.2, you need to configure the option manually.
```
Comment (lichunzhu) [inline dev/reference/tools/data-migration/deploy.md]:
```suggestion
    extra-args: "-B test_db -T test_table"  # The extra Mydumper argument. Starting from DM 1.0.2, DM automatically generates the `tables-list` option. For versions earlier than v1.0.2, you need to configure the option manually.
```

File: dev/how-to/get-started/data-migration.md [hunk 3]
```diff
@@ -245,7 +245,9 @@ password = ""
 port = 3307
 ```
 
-The `flavor` option should be set to `"mysql"` (the default value, and 5.5 < MySQL versions < 8.0 are supported) if migrating from MySQL Server, Percona Server, Percona XtraDB Cluster, or Amazon Aurora or RDS. If migrating from MariaDB Server or MariaDB (Galera) Cluster, use `flavor = "mariadb"` (only MariaDB versions greater than 10.1.2 are supported).
+- If you migrate data from MySQL Server, Percona Server, Percona XtraDB Cluster, Amazon Aurora or RDS, set the `flavor` option to `"mysql"`, which is the default value. For this default value, MySQL versions between 5.5 (not included) and 8.0 (not included) are supported.
```
Comment (yikeke) [inline dev/how-to/get-started/data-migration.md]:
```suggestion
- If you migrate data from MySQL Server, Percona Server, Percona XtraDB Cluster, Amazon Aurora or RDS, set the `flavor` option to `"mysql"`, which is the default value. This value is valid only when you are using a MySQL version between 5.5 (not included) and 8.0 (not included).
```
I rewrote the sentence to make it clearer. Is this what you mean? @TomShawn @csuzhangxc

File: dev/how-to/get-started/data-migration.md [hunk 4]
```diff
@@ -245,7 +245,9 @@ password = ""
...
+- If you migrate data from MariaDB Server or MariaDB (Galera) Cluster, set `flavor = "mariadb"`, which only supports MariaDB version later than 10.1.2.
```
Comment (yikeke) [inline dev/how-to/get-started/data-migration.md]:
```suggestion
- If you migrate data from MariaDB Server or MariaDB (Galera) Cluster, set `flavor = "mariadb"`. You can set this value only when you are using a MariaDB version later than 10.1.2.
```
Same as above.

File: dev/how-to/get-started/data-migration.md [hunk 5]
```diff
@@ -245,7 +245,9 @@ password = ""
...
+- Starting from DM 1.0.2, DM automatically generates the `flavor` and `server-id` options. You do not need to manually configure these options in normal situations.
```
Comment (yikeke) [inline dev/how-to/get-started/data-migration.md]:
```suggestion
- Starting with DM 1.0.2, DM automatically generates values for the `flavor` and `server-id` options. You do not need to manually configure these options in normal situations.
```
Please confirm with the author on "which" DM automatically generates - "the values of the options" or "the options + their values". @TomShawn @csuzhangxc

File: dev/how-to/migrate/from-aurora.md [hunk 6]
```diff
@@ -126,7 +126,7 @@ black-white-list:
 # Mydumper global configuration. Each instance can quote it by the configuration item name.
 mydumpers:
   global:
-    extra-args: "-B test_db -T test_table"  # Only outputs the `test_table` table of the `test_db` schema and can configure any parameter of Mydumper.
+    extra-args: "-B test_db -T test_table"  # The extra Mydumper argument. Since DM 1.0.2, DM automatically generates the "tables-list" option. For versions earlier than 1.0.2, you need to configure this option manually.
```
Comment (yikeke) [inline dev/how-to/migrate/from-aurora.md]: The original Chinese is "table-list 配置" and [one DM-related document](https://pingcap.com/docs-cn/stable/reference/tools/mydumper/#%E4%BD%BF%E7%94%A8-loader-%E6%81%A2%E5%A4%8D-mydumper-%E5%A4%87%E4%BB%BD%E5%87%BA%E6%9D%A5%E7%9A%84%E6%95%B0%E6%8D%AE%E6%97%B6%E6%8A%A5%E9%94%99-invalid-mydumper-files-for-there-are-no--schema-createsql-files-found%E5%BA%94%E8%AF%A5%E5%A6%82%E4%BD%95%E8%A7%A3%E5%86%B3) uses "tables-list". Please confirm with the author on which is correct. @TomShawn @csuzhangxc

File: dev/reference/tools/data-migration/deploy.md [hunk 7]
```diff
@@ -105,7 +105,7 @@ black-white-list:
 mydumpers:
   global:
     mydumper-path: "./bin/mydumper"   # The file path of the Mydumper binary.
-    extra-args: "-B test_db -T test_table"  # Only dumps the "test_table" table of the "test_db" database. It can configure any Mydumper argument.
+    extra-args: "-B test_db -T test_table"  # The extra Mydumper argument. Since DM 1.0.2, DM automatically generates the "tables-list" option. For versions earlier than 1.0.2, you need to configure this option manually.
```
Comment (yikeke) [inline dev/reference/tools/data-migration/deploy.md]: ditto

File: dev/reference/tools/data-migration/query-status.md [hunk 8]
```diff
@@ -6,10 +6,77 @@ category: reference
...
+
+```bash
+» query-status
+```
+
+```
+{
+    "result": true, # Whether the query is successful.
+    "msg": "",      # Describes the cause of the unsuccessful query.
+    "tasks": [      # Replication task list.
```
Comment (yikeke) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
    "tasks": [      # Migration task list.
```

File: dev/reference/tools/data-migration/query-status.md [hunk 9]
```diff
@@ -6,10 +6,77 @@ category: reference
...
+        {
+            "taskName": "test-1",           # The task name
+            "taskStatus": "Running",        # The status of the running task, including "New", "Running", "Paused", "Stopped", "Finished", and "Error".
```
Comment (yikeke) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
            "taskStatus": "Running",        # The status of the task, including "New", "Running", "Paused", "Stopped", "Finished", and "Error".
```

File: dev/reference/tools/data-migration/query-status.md [hunk 10]
```diff
@@ -6,10 +6,77 @@ category: reference
...
+            "workers": [                    # DM-worker list.
```
Comment (yikeke) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
            "workers": [                    # The list of DM-workers that are used by the task.
```

File: dev/reference/tools/data-migration/query-status.md [hunk 11]
```diff
@@ -6,10 +6,77 @@ category: reference
...
+            "workers": [
+                "127.0.0.1:8263",
+                "127.0.0.1:8264"
+            ]
+        }
+    ]
+}
+```
+
+For detailed descriptions of `taskStatus` in a task, refer to [Task status](#task-status).
```
Comment (yikeke) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
For detailed descriptions of `taskStatus` under the `tasks` section, refer to [Task status](#task-status).
```

File: dev/reference/tools/data-migration/query-status.md [hunk 12]
```diff
@@ -6,10 +6,77 @@ category: reference
...
+
+It is recommended that you use `query-status` by following these steps:
```
Comment (yikeke) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
It is recommended that you use `query-status` by the following steps:
```

File: dev/reference/tools/data-migration/query-status.md [hunk 13]
```diff
@@ -6,10 +6,77 @@ category: reference
...
+
+1. Use `query-status` to check whether each on-going task is in the normal state.
+2. If any error occurs in a task, use the `query-status <taskName>` command to see detailed error information. `<taskName>` in this command indicates the name of the task that encounters the error.
+
+## Task status
+
+The status of a DM replication task depends on the status of each subtask assigned to DM-worker. For detailed descriptions of subtask status, see [Subtask status](#subtask-status). The table below shows how the subtask status is related to task status.
```
Comment (yikeke) [inline dev/reference/tools/data-migration/query-status.md]:
```suggestion
The status of a DM migration task depends on the status of each subtask assigned to DM-worker. For detailed descriptions of subtask status, see [Subtask status](#subtask-status). The table below shows how the subtask status is related to task status.
```

File: dev/how-to/get-started/data-migration.md (hunk 5 above)
Comment (TomShawn) [reply dev/how-to/get-started/data-migration.md]: "the values of the options"

File: dev/how-to/migrate/from-aurora.md (hunk 6 above)
Comment (TomShawn) [reply dev/how-to/migrate/from-aurora.md]: `--tables-list` is correct.

**Pull request merged**